sphinx/source/docs/user_guide/concepts.rst

```diff
@@ -135,6 +135,49 @@ Another way to generate various kinds of output is by using Bokeh's powerful
 
 .. _userguide_interfaces:
 
+Bokeh settings
+--------------
+
+There are various global settings that influence how Bokeh operates.
+
+There are several methods you can use to change Bokeh's configuration. Directly
+in the Python code, in a YAML configuration file, or with environment variables,
+for example. A full list of all available settings and how to change them is
+available at :ref:`bokeh.settings`.
+
+Some of most commonly used settings are:
```

Comment:
maybe "useful" is better than "common". Apart from `BOKEH_RESOURCES`, most of these probably get ~no user usage